src/buf.yaml

```diff
@@ -51,6 +51,8 @@ breaking:
     - storage-types/src/connections/string_or_secret.proto
     # reason: no reason specified
     - storage-types/src/sinks.proto
+    # reason: no reason specified
```

Comment:
Does this need a reason?

Reply:
ah yes I'll add one, thought it would pick up my `buf breaking` comment automatically